packages/web/src/scss/components/FileUploader/_FileUploaderInput.scss

```diff
@@ -99,4 +144,18 @@ $_field-name: 'FileUploaderInput';
 .FileUploaderInput.is-dragging .FileUploaderInput__input ~ .FileUploaderInput__dropZone {
     border: theme.$input-drop-zone-dragging-border;
 }
+
+:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled)
+    > .FileUploaderInput__input
+    ~ .FileUploaderInput__dropZone {
+    @include form-fields-tools.box-field-disabled-input-v1();
+}
+
+.FileUploaderInput .FileUploaderInput__input:disabled ~ .FileUploaderInput__dropZone {
+    @include form-fields-tools.box-field-disabled-input-v1();
+}
```

Comment:
Again, the same questions:

- two scenarios for disabled style,
- fragile use of `box-field-disabled-input-v1()` mixin.

Also, I think these styles could live together with the other disabled styles (unless there is a reason, of course).